helix-rest/src/main/java/org/apache/helix/rest/server/ServerContext.java

```diff
@@ -64,6 +66,8 @@ public ServerContext(String zkAddr) {
     // cannot be started correctly.
     _helixDataAccessorPool = new HashMap<>();
     _taskDriverPool = new HashMap<>();
+    // Initiate the singleton ZkMetadataStoreDirectory instance to allow it to be closed later
```

Comment:
Nit: "Initialize" would the the term.

Reply:
I agree.

Reply:
Hm, I think my eyes were caught by the "initiate" in the comment on line 63, therefore I wrote "initiate" too.